### PR TITLE
Update go to 1.19.3

### PR DESCRIPTION
**Pull request by prestonvanloon**
**What type of PR is this?**

Other

**What does this PR do? Why is it needed?**

Updates the go version to 1.19.3

**Which issues(s) does this PR fix?**

**Other notes for review**

There are some changes to gofmt in 1.19 and this PR must include those changes. 
The diff is large, but please scan each file to ensure it is truly only formatting. 

https://tip.golang.org/doc/go1.19

### CODE REVIEW

**Comment by prestonvanloon** [comment]
A note for reviewers: You can checkout this branch then use `git diff -w origin/develop` to see the diff without whitespace formatting diff.

**Review by saolyn — COMMENTED** [review]
There seem to be quite a few additional new lines that make the formatting messier rather than cleaner.

File: beacon-chain/core/blocks/header.go [hunk 1]
```diff
@@ -18,27 +18,27 @@ import (
...
-//    # Cache current block as the new latest block
-//    state.latest_block_header = BeaconBlockHeader(
-//        slot=block.slot,
-//        proposer_index=block.proposer_index,
-//        parent_root=block.parent_root,
-//        state_root=Bytes32(),  # Overwritten in the next process_slot call
-//        body_root=hash_tree_root(block.body),
-//    )
+//	def process_block_header(state: BeaconState, block: BeaconBlock) -> None:
+//	  # Verify that the slots match
```
[the remark this answers is not in the document]
Comment (terencechain) [reply beacon-chain/core/blocks/header.go:22]: I think go fmt automatically does this

File: beacon-chain/core/signing/signing_root.go [hunk 2]
```diff
@@ -37,14 +37,15 @@ func ComputeDomainAndSign(st state.ReadOnlyBeaconState, epoch types.Epoch, obj f
...
-//    """
-//    Return the signing root for the corresponding signing data.
-//    """
-//    return hash_tree_root(SigningData(
-//        object_root=hash_tree_root(ssz_object),
-//        domain=domain,
-//    ))
+//
+//		def compute_signing_root(ssz_object: SSZObject, domain: Domain) -> Root:
+//	   """
```
[the remark this answers is not in the document]
Comment (terencechain) [reply beacon-chain/core/signing/signing_root.go:42]: I'm not sure if go fmt automatically does this, if it does then maybe we should just remove the spec code comments all at once

**Review by terencechain — APPROVED** [review]

**Pull request merged**